misk/testing/src/main/kotlin/misk/testing/MockTracingBackendModule.kt

```diff
@@ -0,0 +1,11 @@
+package misk.testing
+
+import io.opentracing.Tracer
+import io.opentracing.mock.MockTracer
+import misk.inject.KAbstractModule
+
+class MockTracingBackendModule : KAbstractModule() {
+  override fun configure() {
+    bind(Tracer::class.java).to(MockTracer::class.java).asEagerSingleton()
```

Comment:
are we going with the term `Fake` or `Mock`? seems like `Fake` is used the most

Reply:
Meh, was just keeping the module name consistent with the `MockTracer` provided by the opentracing library. We can just rename the module in the future if it becomes confusing.

Reply:
I believe conceptually a Fake is something that implements an interface in-memory and serves a variety of purposes, while a Mock implements some fraction of the API and has canned responses to specific inputs.

Reply:
Agreed, and MockTracer functions very much like a Mock in that it does nothing except track calls to methods.